internal/rollup/postinstall-patches.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * @license
+ * Copyright 2017 The Bazel Authors. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+try {
+  require.resolve('shelljs');
```

Comment:
let's not add a shelljs dependency - we are making *everyone* who uses rules_nodejs download this, right? or at least anyone who loads from the rollup rule. We can probably just do this with node APIs